Sources/Sentry/SentryReplayApi.m

```diff
@@ -39,20 +39,26 @@ - (void)resume
 
 - (void)start
 {
-    SentrySessionReplayIntegration *replayIntegration
-        = (SentrySessionReplayIntegration *)[SentrySDK.currentHub
-            getInstalledIntegration:SentrySessionReplayIntegration.class];
+    // Start could be misused and called multiple times, causing it to
+    // be initialized more than once before being installed.
+    // Synchronizing it will prevent this problem.
+    @synchronized(self) {
```

Comment:
`m`: I guess a double-checked lock would make sense here to avoid acquiring the lock in 99.99% of the cases after the first start call. We use it in the dependency container. Please use `SENTRY_DISABLE_THREAD_SANITIZER` to disable the thread sanitizer warning.